### PR TITLE
Node geometry

### PR DESCRIPTION
**Pull request by deltakosh**

### CODE REVIEW

**Comment by bjsplat** [comment]
Please make sure to label your PR with "bug", "new feature" or "breaking change" label(s).
To prevent this PR from going to the changelog marked it with the "skip changelog" label.

**Comment by bjsplat** [comment]
Reviewer - this PR has made changes to one or more package.json files.

**Comment by bjsplat** [comment]
Please make sure to label your PR with "bug", "new feature" or "breaking change" label(s).
To prevent this PR from going to the changelog marked it with the "skip changelog" label.

**Comment by bjsplat** [comment]
Reviewer - this PR has made changes to one or more package.json files.

File: packages/public/umd/babylonjs-node-editor/webpack.config.js
Comment (RaananW) [inline packages/public/umd/babylonjs-node-editor/webpack.config.js:1]: i believe this is not the file you meant to commit? this is the node-editor. There should be a directory for the node-geometry-editor in public UMD

**Comment by bjsplat** [comment]
Please make sure to label your PR with "bug", "new feature" or "breaking change" label(s).
To prevent this PR from going to the changelog marked it with the "skip changelog" label.

**Comment by bjsplat** [comment]
Please make sure to label your PR with "bug", "new feature" or "breaking change" label(s).
To prevent this PR from going to the changelog marked it with the "skip changelog" label.

**Comment by bjsplat** [comment]
Reviewer - this PR has made changes to one or more package.json files.

File: packages/public/umd/babylonjs-node-editor/webpack.config.js
Comment (deltakosh) [reply packages/public/umd/babylonjs-node-editor/webpack.config.js:1]: oh crap!

**Comment by bjsplat** [comment]
Please make sure to label your PR with "bug", "new feature" or "breaking change" label(s).
To prevent this PR from going to the changelog marked it with the "skip changelog" label.

**Comment by bjsplat** [comment]
Reviewer - this PR has made changes to one or more package.json files.

**Comment by bjsplat** [comment]
Snapshot stored with reference name:
refs/pull/14141/merge

Test environment: 
https://babylonsnapshots.z22.web.core.windows.net/refs/pull/14141/merge/index.html

To test a playground add it to the URL, for example:

https://babylonsnapshots.z22.web.core.windows.net/refs/pull/14141/merge/index.html#WGZLGJ#4600

Links to test babylon tools with this snapshot:

https://playground.babylonjs.com/?snapshot=refs/pull/14141/merge
https://sandbox.babylonjs.com/?snapshot=refs/pull/14141/merge
https://gui.babylonjs.com/?snapshot=refs/pull/14141/merge
https://nme.babylonjs.com/?snapshot=refs/pull/14141/merge

To test the snapshot in the playground with a playground ID add it after the snapshot query string:

https://playground.babylonjs.com/?snapshot=refs/pull/14141/merge#BCU1XR#0

**Comment by bjsplat** [comment]
Reviewer - this PR has made changes to one or more package.json files.

**Comment by bjsplat** [comment]
Reviewer - this PR has made changes to one or more package.json files.

**Comment by bjsplat** [comment]
Reviewer - this PR has made changes to one or more package.json files.

File: packages/public/umd/babylonjs-node-geometry-editor/src/index.ts [hunk 1]
```diff
@@ -0,0 +1,3 @@
+import * as nodeEditor from "node-editor/legacy/legacy";
```
Comment (RaananW) [inline packages/public/umd/babylonjs-node-geometry-editor/src/index.ts:1]: should be node-geometry-editor

**Comment by bjsplat** [comment]
Reviewer - this PR has made changes to one or more package.json files.

**Comment by bjsplat** [comment]
Reviewer - this PR has made changes to one or more package.json files.

**Comment by bjsplat** [comment]
Visualization tests for webgl1 have failed. If some tests failed because the snapshots do not match, the report can be found at

https://babylonsnapshots.z22.web.core.windows.net/refs/pull/14141/merge/testResults/webgl1/index.html

If tests were successful afterwards, this report might not be available anymore.

**Comment by bjsplat** [comment]
Visualization tests for webgl2 have failed. If some tests failed because the snapshots do not match, the report can be found at

https://babylonsnapshots.z22.web.core.windows.net/refs/pull/14141/merge/testResults/webgl2/index.html

If tests were successful afterwards, this report might not be available anymore.

**Comment by bjsplat** [comment]
Visualization tests for webgl1 have failed. If some tests failed because the snapshots do not match, the report can be found at

https://babylonsnapshots.z22.web.core.windows.net/refs/pull/14141/merge/testResults/webgl1/index.html

If tests were successful afterwards, this report might not be available anymore.

**Comment by bjsplat** [comment]
Visualization tests for webgl2 have failed. If some tests failed because the snapshots do not match, the report can be found at

https://babylonsnapshots.z22.web.core.windows.net/refs/pull/14141/merge/testResults/webgl2/index.html

If tests were successful afterwards, this report might not be available anymore.

**Comment by bjsplat** [comment]
Visualization tests for webgl1 have failed. If some tests failed because the snapshots do not match, the report can be found at

https://babylonsnapshots.z22.web.core.windows.net/refs/pull/14141/merge/testResults/webgl1/index.html

If tests were successful afterwards, this report might not be available anymore.

**Comment by bjsplat** [comment]
Visualization tests for webgl2 have failed. If some tests failed because the snapshots do not match, the report can be found at

https://babylonsnapshots.z22.web.core.windows.net/refs/pull/14141/merge/testResults/webgl2/index.html

If tests were successful afterwards, this report might not be available anymore.

**Comment by bjsplat** [comment]
Reviewer - this PR has made changes to one or more package.json files.

**Comment by bjsplat** [comment]
Reviewer - this PR has made changes to one or more package.json files.

**Comment by bjsplat** [comment]
Reviewer - this PR has made changes to one or more package.json files.

**Comment by bjsplat** [comment]
Reviewer - this PR has made changes to one or more package.json files.

**Comment by bjsplat** [comment]
Reviewer - this PR has made changes to one or more package.json files.

**Comment by bjsplat** [comment]
Reviewer - this PR has made changes to one or more package.json files.

**Review by Popov72 — APPROVED** [review]

**Pull request merged**